Error compiling Cython file:
------------------------------------------------------------
...
        self.numcolbins=bin_col_vals.numbins
        self.num_datapts=<int>row_index.size
        self.col_view=col_vals
        self.row_index_view=row_index
        self.col_index_view=col_index
        self.thisptr = new Histo(self.numrowbins,self.numcolbins,self.num_datapts, 
                          ^
------------------------------------------------------------

C:\Users\dashamstyr\Dropbox\Classes\EOS582\shared_code\satellite\fasthist.pyx:42:27: Operation only allowed in c++

Error compiling Cython file:
------------------------------------------------------------
...
        grid_mean[:,:]=np.nan
        self.thisptr.get_mean(<float*> np.PyArray_DATA(data_vec),<float*> np.PyArray_DATA(grid_mean))
        return grid_mean
        
    def __dealloc__(self):
        del self.thisptr
       ^
------------------------------------------------------------

C:\Users\dashamstyr\Dropbox\Classes\EOS582\shared_code\satellite\fasthist.pyx:65:8: Operation only allowed in c++
fasthist.c
C:\Users\dashamstyr\.pyxbld\temp.win32-2.7\Release\pyrex\fasthist.c(1) : fatal error C1189: #error :  Do not use this file, it is the result of a failed Cython compilation.
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\Python27\lib\site-packages\pyximport\pyximport.py", line 431, in load_module
    language_level=self.language_level)
  File "C:\Python27\lib\site-packages\pyximport\pyximport.py", line 209, in load_module
    inplace=build_inplace, language_level=language_level)
  File "C:\Python27\lib\site-packages\pyximport\pyximport.py", line 186, in build_module
    reload_support=pyxargs.reload_support)
  File "C:\Python27\lib\site-packages\pyximport\pyxbuild.py", line 104, in pyx_to_dll
    dist.run_commands()
  File "C:\Python27\lib\distutils\dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "C:\Python27\lib\distutils\dist.py", line 972, in run_command
    cmd_obj.run()
  File "C:\Python27\lib\site-packages\Cython\Distutils\build_ext.py", line 163, in run
    _build_ext.build_ext.run(self)
  File "C:\Python27\lib\distutils\command\build_ext.py", line 339, in run
    self.build_extensions()
  File "C:\Python27\lib\site-packages\Cython\Distutils\build_ext.py", line 171, in build_extensions
    self.build_extension(ext)
  File "C:\Python27\lib\distutils\command\build_ext.py", line 498, in build_extension
    depends=ext.depends)
  File "C:\Python27\lib\distutils\msvc9compiler.py", line 548, in compile
    raise CompileError(msg)
ImportError: Building module fasthist failed: ['CompileError: command \'"C:\\Program Files (x86)\\Microsoft Visual Studio 9.0\\VC\\BIN\\cl.exe"\' failed with exit status 2\n']